app/README.md

```diff
@@ -78,14 +78,14 @@ Our web server is now running on **port 8000**, you can access it by going to [h
     <h3>Geolocation integration</h3>
     <div id="map"></div>
     <script async defer
-    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDvG9inIBgoXyFWZAhZqyzyJub_7VXsl8c&callback=initMap">
+    src="https://maps.googleapis.com/maps/api/js?key=xxxxxxxxxxxxxxxxxxx&callback=initMap">
```

Comment:
Ha, do revoke that API key. It will be in git history forever

Reply:
It's done, I revoked it 👍